src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Core/Filters/AuthorizePartyUuidClaimFilter.cs

```diff
@@ -0,0 +1,51 @@
+using Altinn.AccessManagement.Core.Constants;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.AccessManagement.Core.Filters;
+
+/// <summary>
+/// Verifies that user uuid claim exists in token and adds the value to http context items bag.
+/// If claim is not present it will return cancel pipeline and return 401 unauthorized.
+/// </summary>
+public partial class AuthorizePartyUuidClaimFilter(ILogger<AuthorizePartyUuidClaimFilter> logger) : IAuthorizationFilter
+{
+    private ILogger<AuthorizePartyUuidClaimFilter> Logger { get; } = logger;
+
+    /// <inheritdoc/>
+    public void OnAuthorization(AuthorizationFilterContext context)
+    {
+        if (context.HttpContext.User?.Identity?.IsAuthenticated != true)
+        {
+            Log.ThereIsNoAuthorizedUser(Logger);
+            context.Result = new UnauthorizedResult();
+            return;
+        }
+
+        var claim = context.HttpContext.User?.Claims?
+            .FirstOrDefault(c => c.Type.Equals(AltinnCoreClaimTypes.PartyUuid, StringComparison.OrdinalIgnoreCase));
+
+        if (claim != null && Guid.TryParse(claim.Value, out Guid userUuid))
+        {
+            if (!context.HttpContext.Items.ContainsKey("user_uuid"))
+            {
+                context.HttpContext.Items.Add("user_uuid", userUuid);
```

Comment:
ahh, her du setter det. Hva er egentlig verdien i å flytte det fra user claims til items bare for å lese det ut på nytt der i fra?

Reply:
ignorerte å legge useruuid til i Items. Utførte LINQ query på nytt i extensions